README.md

```diff
@@ -152,3 +152,38 @@ images:
     repository: rancher/fluent-bit
     tag: 1.7.4
 ```
+
+#### Pull Request rules
+
+Please create your Pull Request title following this rule:
+
+```
+[dev-v2.X] <chart> <version> <action>
+[release-v2.X] <chart> <version> <action>
+```
+
+A working example:
+```
+[dev-v2.8] rancher-istio 103.2.0+up1.19.6 update
+```
+
+- `<chart>`: The full name of the charts exactly how it is written under `/charts folder`
+- `<version>`: The full version of the chart, exactly how it is written under `release.yaml`
+- `<action>`: `update`; `remove`; `add`
+
+What you should keep in mind for releasing charts:
```

Comment:
Are we going to add this responsibility to the teams now?

Reply:
The teams must be aware of these rules at the time of the chart update so we don't have problems in the release merge final process.

Reply:
Agreed.